UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentAncillaryFilesTest.cs

```diff
@@ -348,5 +349,39 @@ public async Task WhenValidCreateLargeMediaSerialEncFileRequest_ThenReturnTrueRe
             Assert.AreEqual(true, response);
         }
         #endregion
+
+        #region CreateLargeExchangeSetCatalogFile
+        [Test]
+        public async Task WhenValidCreateLargeExchangeSetCatalogFileRequest_ThenReturnTrueReponse()
+        {
+            var b1 = A.Fake<IDirectoryInfo>();
```

Comment:
What's the point of the b1 variable? It's just reassigned to directoryInfos and then that variable is used. Why not just set directoryInfos  = A.Fake<IDirectoryInfo>();

Reply:
Resolved